From 062434ab3be76d4fa5973bb199ccfd5b68c11720 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Tue, 23 Jul 2024 11:21:47 +0100
Subject: [PATCH 1198/1215] dts: rp1: restrict i2s burst lengths to 4

The associated DMAC has channels that do not support longer bursts.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 arch/arm64/boot/dts/broadcom/rp1.dtsi | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/arm64/boot/dts/broadcom/rp1.dtsi
+++ b/arch/arm64/boot/dts/broadcom/rp1.dtsi
@@ -400,6 +400,7 @@
 			#sound-dai-cells = <0>;
 			dmas = <&rp1_dma RP1_DMA_I2S0_TX>,<&rp1_dma RP1_DMA_I2S0_RX>;
 			dma-names = "tx", "rx";
+			dma-maxburst = <4>;
 			status = "disabled";
 		};
 
@@ -413,6 +414,7 @@
 			#sound-dai-cells = <0>;
 			dmas = <&rp1_dma RP1_DMA_I2S1_TX>,<&rp1_dma RP1_DMA_I2S1_RX>;
 			dma-names = "tx", "rx";
+			dma-maxburst = <4>;
 			status = "disabled";
 		};
 
